core/modules/filters/contains.js

```diff
@@ -0,0 +1,77 @@
+/*\
+title: $:/core/modules/filters/contains.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator for comparing the contents of an input tiddlers field and the
+operand.
+
+This filter returns the input title iff the operand is a subset of the given
+field of the input.
```

Comment:
iff .. typo?

Reply:
no, iff as in 'if and only if', should I avoid using language like that in the documentation?

Reply:
If it isn't proper english ... yes

Reply:
no documentation in the modules, pls ...and no dev / tech lingo (iff)